packages/builder/src/contract/contracts/Xtokens/Xtokens.ts

```diff
@@ -6,9 +6,11 @@ import { decodeAddress } from '@polkadot/util-crypto';
 import { ContractConfigBuilder } from '../../ContractBuilder.interfaces';
 import { ContractConfig } from '../../ContractConfig';
 
+const U_64_MAX = 18446744073709551615n;
+
 export function Xtokens() {
   return {
-    transfer: (weight = 4_000_000_000): ContractConfigBuilder => ({
+    transfer: (weight = U_64_MAX): ContractConfigBuilder => ({
```

Comment:
https://github.com/moonbeam-foundation/moonbeam/blob/aa47060cc9cce9b9e77ea75ec77b38696d647ebd/precompiles/xtokens/src/lib.rs#L99C7-L99C7
we pass rust U64:MAX for the precompile to consider it as unlimited weight